### PR TITLE
Update README install instructions

### PR DESCRIPTION
**Pull request by iriberri**
cc @ethervoid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,21 +57,23 @@ Steps to deploy a new Data Services API version :
...
 
     ```
-    # select CDB_Conf_SetConf('redis_metadata_config', '{"sentinel_host": "localhost", "sentinel_port": 26379, "sentinel_master_id": "mymaster", "timeout": 0.1, "redis_db": 5}');
-    # select CDB_Conf_SetConf('redis_metrics_config', '{"sentinel_host": "localhost", "sentinel_port": 26379, "sentinel_master_id": "mymaster", "timeout": 0.1, "redis_db": 5}');
+    # SELECT CDB_Conf_SetConf('redis_metadata_config', '{"sentinel_host": "localhost", "sentinel_port": 26379, "sentinel_master_id": "mymaster", "timeout": 0.1, "redis_db": 5}');
+    # SELECT CDB_Conf_SetConf('redis_metrics_config', '{"sentinel_host": "localhost", "sentinel_port": 26379, "sentinel_master_id": "mymaster", "timeout": 0.1, "redis_db": 5}');
     
-    # select CDB_Conf_SetConf('heremaps_conf', '{"app_id": "APP_ID", "app_code": "APP_CODE"}');
-    # select CDB_Conf_SetConf('user_config', '{"is_organization": false, "entity_name": "<YOUR_USERNAME>"}')
+    # SELECT CDB_Conf_SetConf('heremaps_conf', '{"app_id": "APP_ID", "app_code": "APP_CODE"}');
```
Comment (ethervoid) [inline README.md]: Here we have to add the `geocoder_cost_per_hit` and `SELECT CDB_Conf_SetConf('logger_con', '{"geocoder_log_path": "/tmp/geocodings.log"}')

File: README.md [hunk 2]
```diff
@@ -57,21 +57,23 @@ Steps to deploy a new Data Services API version :
...
+    # SELECT CDB_Conf_SetConf('user_config', '{"is_organization": false, "entity_name": "<YOUR_USERNAME>"}')
+    # SELECT CDB_Conf_SetConf('mapzen_conf', '{"routing_app_key": "ROUTING_API_KEY", "geocoder_app_key": "GEOCODER_API_KEY"}');
+
```
Comment (ethervoid) [inline README.md]: Add `SELECT CDB_Conf_SetConf('logger_con', '{"geocoder_log_path": "/tmp/geocodings.log"}')`

**Comment by iriberri** [comment]
I'm seeing that redis conf is also missing redis_host and redis_port at least.

**Comment by iriberri** [comment]
@ethervoid I just added the sentinel/redis differences we talked about.

**Comment by ethervoid** [comment]
retest this please

**Comment by ethervoid** [comment]
:shipit:

**Pull request merged**